CoreScriptsRoot/LoadingScript.lua

```diff
@@ -5,89 +5,84 @@
 -- Constants
 local PLACEID = game.PlaceId
 
-local MPS = game:GetService('MarketplaceService')
-local UIS = game:GetService('UserInputService')
-local guiService = game:GetService("GuiService")
+local MarketplaceService = game:GetService('MarketplaceService')
+local InputService = game:GetService('UserInputService')
+local guiService = game:GetService('GuiService')
 local ContextActionService = game:GetService('ContextActionService')
-local RobloxGui = game:GetService("CoreGui"):WaitForChild("RobloxGui")
+local ReplicatedFirst = game:GetService('ReplicatedFirs')
```

Comment:
Does `ReplicatedFirs` actually get the right service? It sort of feels like this code has not been tested.